scss/utils.scss

```diff
@@ -1,12 +1,35 @@
 $navbar-height: 70px;
 
-@mixin detail-attribute {
+@mixin bold-text($font-size) {
   font-weight: bold;
-  font-size: 18px;
+  font-size: $font-size;
 }
 
 @mixin box-shadow {
   -webkit-box-shadow: 2px 4px 8px 1px rgba(122,117,122,1);
   -moz-box-shadow: 2px 4px 8px 1px rgba(122,117,122,1);
   box-shadow: 2px 4px 8px 1px rgba(122,117,122,1);
 }
+
+@mixin logo-size{
+  width: 100%;
+  max-width: 212px;
+}
+
+@mixin general-btn-style($text-color, $bg-color) {
+  @include form-element;
+  color: $text-color;
+  background-color: $bg-color;
+}
+
+@mixin fill-green-btn {
```

Comment:
why a mixin and not a class? both are ok but a class can be used directly on the template while a mixin not, you have to define a class and make that class extend from the mixin.